integration/spark/integrations/spark3/container/pysparkRddToTableCompleteEvent.json

```diff
@@ -0,0 +1,84 @@
+{
+  "eventType": "COMPLETE",
+  "run": {
+    "facets": {
+    }
+  },
+  "job": {
+    "namespace": "testPysparkRddToTable",
+    "name": "spark_rdd_to_table.execute_insert_into_hadoop_fs_relation_command"
+  },
+  "inputs": [
+    {
+      "namespace": "file",
+      "name": "/test_data/rdd_to_csv_output",
+      "facets": {
+        "schema": {
+          "_producer": "https://github.com/OpenLineage/OpenLineage/tree/$VERSION/integration/spark",
+          "fields": [
+            {
+              "name": "name",
+              "type": "string"
+            },
+            {
+              "name": "age",
+              "type": "integer"
+            }
+          ],
+          "_schemaURL": "https://openlineage.io/spec/1-0-1/OpenLineage.json#/definitions/SchemaDatasetFacet"
+        },
+        "dataSource": {
+          "_producer": "https://github.com/OpenLineage/OpenLineage/tree/$VERSION/integration/spark",
+          "name": "file",
+          "uri": "file",
+          "_schemaURL": "https://openlineage.io/spec/1-0-1/OpenLineage.json#/definitions/DatasourceDatasetFacet"
+        }
+      }
+    }
+  ],
+  "outputs": [
+    {
+      "namespace": "file",
+      "name": "/test_data/rdd_to_table",
+      "facets": {
+        "schema": {
+          "_producer": "https://github.com/OpenLineage/OpenLineage/tree/$VERSION/integration/spark",
+          "fields": [
+            {
+              "name": "name",
+              "type": "string"
+            },
+            {
+              "name": "age",
+              "type": "integer"
+            }
+          ],
+          "_schemaURL": "https://openlineage.io/spec/1-0-1/OpenLineage.json#/definitions/SchemaDatasetFacet"
+        },
+        "dataSource": {
+          "_producer": "https://github.com/OpenLineage/OpenLineage/tree/$VERSION/integration/spark",
+          "name": "file",
+          "uri": "file",
+          "_schemaURL": "https://openlineage.io/spec/1-0-1/OpenLineage.json#/definitions/DatasourceDatasetFacet"
+        },
+        "stats" : {
+          "_producer" : "https://github.com/OpenLineage/OpenLineage/tree/$VERSION/integration/spark",
+          "_schemaURL" : "https://github.com/OpenLineage/OpenLineage/tree/$VERSION/integration/spark/facets/spark-2.4/v1/output-statistics-facet.json",
+          "rowCount" : 0,
+          "size" : -1,
```

Comment:
Are the output stats the only things that differ between the Spark 2 and Spark 3 output? Can we just ignore or find/replace these values in the string content before we do the comparison? Similar to how we find/replace the schema version or ignore the run_id in other tests?

Reply:
there are few additional fields, but the problem with `mockserver` matcher, I'll take a look can we somehow ignore fields there

Reply:
```
"schema": [
              {
                "name": "count",
                "dataType": {
                  "numeric": {},
                  "integral": {},
                  "ordering": {}
                },
                "nullable": false,
                "metadata": {
                  "map": {}
                },
                "comment": null
              }
            ],
```

vs
```

 "schema": {
              "type": "struct",
              "fields": [
                {
                  "name": "count",
                  "type": "long",
                  "nullable": false,
                  "metadata": {}
                }
              ]
            },
```

Reply:
we can use placeholders in mock server
https://www.mock-server.com/mock_server/creating_expectations.html#button_match_request_by_body_with_json_placeholders